leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/LeshanClusterServer.java

```diff
@@ -77,83 +79,93 @@ public static void main(String[] args) {
 
         // Abort if unexpected options
         if (cl.getArgs().length > 0) {
-            System.out.println("Unexpected option or arguments : " + cl.getArgList());
+            System.err.println("Unexpected option or arguments : " + cl.getArgList());
             formatter.printHelp(USAGE, null, options, FOOTER);
             return;
         }
 
         // get cluster instance Id
-        String clusterInstanceId = System.getenv("INSTANCEID");
```

Comment:
If we want to remove it. We should also remove it in bsserver-demo and server-demo. We should also change the script on the sandbox which use this envvar.